### ISSUE TITLE
[$15]{MSFT-91} {Member} Focus element is not element when keyboard is used.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2800]
**Steps to Reproduce**
1. Login to https://accounts.topcoder.com/member.
2. Use Tab to navigate in the page.

**Expected Result**
Focus element should be highlighted.

2.4.7 Intent

The purpose of this success criterion is to help a person know which element has the keyboard focus.

The purpose of this success criterion is to help a person know which element among multiple elements has the keyboard focus. If there is only one keyboard actionable control on the screen, the success criterion would be met because the visual design presents only one keyboard actionable item.

**Actual Result**
Focus element is not getting highlighted.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: AA
**WCAG Success Criterion**: 2.4.7 Focus Visible
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=4c403434%2D55d3%2D4752%2Db379%2D635595c2fa3a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fjackgid16%2Daccessibility%2F2

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096273 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096273 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096273 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
PR at https://github.com/appirio-tech/accounts-app/pull/243

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097300 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097300 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097300 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu issue https://github.com/topcoder-platform/community-app/issues/2875 was marked as duplicate of this issue. The prize money of issue #2875 will be added to this issue, right?

**Comment by veshu** [comment]
@r0hit-gupta yes let reviewer confirm that, I would definitely like to keep separate until it goes to QA reviewer. Just remind me later during payment if this missed.

**Comment by mishacucicea** [comment]
@r0hit-gupta The SSO links are focusable, privacy policy are already done.
You need to take care of `Forgot your password?` link, `LOG IN` button and `Join Now` Link. I can give it a prize boost.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097300 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@mishacucicea done. Please review.

**Comment by mishacucicea** [comment]
@r0hit-gupta Did you took care of the other focusable items discussed above?

**Comment by r0hit-gupta** [comment]
@mishacucicea yes I did.

**Comment by mishacucicea** [comment]
@r0hit-gupta I see just the .privacy-policy class changed, is the adjustment somewhere else?

**Comment by r0hit-gupta** [comment]
@mishacucicea I did not get you. In what way it has changed? I haven't done any kind of adjustment.

**Comment by mishacucicea** [comment]
@r0hit-gupta  Looking again over the code changes, I don't see how `Forgot your password?` link, `LOG IN` button and `Join Now` Link were fixed, so testing might reject your fix.

**Comment by tosha5252** [comment]
Forgot your password, Privacy Policy and Join Now link are still not in focus when tabbing.

Env:
Windows 10
Browser: Chrome

**Comment by r0hit-gupta** [comment]
@tosha5252 @mishacucicea working for me on local.

@mishacucicea here are the code changes.

Privacy Policy - https://github.com/appirio-tech/accounts-app/pull/243

Forgot your password - https://github.com/r0hit-gupta/accounts-app/blob/2c4432ef75e27e9512ff4f6e558345c0a3fa0cea/app/styles/tc/login.scss#L35

Join Now - https://github.com/r0hit-gupta/accounts-app/blob/2c4432ef75e27e9512ff4f6e558345c0a3fa0cea/app/styles/tc/account.scss#L200

Login button - https://github.com/r0hit-gupta/accounts-app/blob/2c4432ef75e27e9512ff4f6e558345c0a3fa0cea/app/styles/tc/login.scss#L14

**Comment by nithyaasworld** [comment]
Verified on production.

**Comment by crazyk07** [comment]
Closing for Payment

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097300<br/><br/>```This is an automated message for crazyk via Topcoder X```
